Add Entry: Java Type-Conversion (#635)
Codecademy/docs: content/java/concepts/type-conversion/type-conversion.md

---
Title: 'Type Conversion'

> Description: 'Type conversion is the process where a value of one data type is converted to a value of another data type.'

Reviewer: ```suggestion
Description: 'Type conversion is the process of converting a value of one data type into another data type.'
```
